hedgehog/src/Hedgehog/Internal/TH.hs

```diff
@@ -13,6 +14,9 @@ import           Hedgehog.Internal.Discovery
 import           Hedgehog.Internal.Property
 
 import           Language.Haskell.TH
+#if MIN_VERSION_template_haskell(2, 12, 0)
+                    hiding (TExpQ)
+#endif
 import           Language.Haskell.TH.Syntax
```

Comment:
This was lazy on my part, they should have been explicit imports from the beginning (then `CPP` wouldn't be required here.)